RyuSocks/Packets/Auth/UsernameAndPassword/UsernameAndPasswordRequest.cs

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (C) RyuSOCKS
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+using System;
+using System.Security.Authentication;
+using System.Text;
+
+namespace RyuSocks.Packets.Auth.UsernameAndPassword
+{
+    public class UsernameAndPasswordRequest : Packet
+    {
+        public UsernameAndPasswordRequest(byte[] packetBytes)
+        {
+            Bytes = packetBytes;
+        }
```

Comment:
Move this below the properties.

I'll eventually add analyzers for this, so it can be done automatically.